itunes-link-engine/trunk/options.php

```diff
@@ -862,5 +892,22 @@ function getGeoriotAffiliates(suppressError) {
         you have added your own via the Geniuslink dashboard.</strong>
       Please <a href="mailto:hi@geni.us">contact Geniuslink</a> if you have any questions.
     </p>
+
+	<h4 id="ile-faq-on-click">How does the on-click feature work?</h4>
+	<p>The “preserve until click time”  option leaves your link URLs alone until the moment a visitor clicks them. This way, when visitors hover over or copy your product links, they 
```

Comment:
Can we make sure that we use " instead of the comma's going up and down since they give issues encoding in some situations

Reply:
I'm not exactly sure what you mean here

Reply:
@aallfredo might be referring to the curly quotes around "preserve until click time". Sometimes copy-pasting from Docs will result in the browser showing junk in the page because it doesn't recognize those..

Reply:
Ah okay gotcha